src/components/TaskForm.tsx

```diff
@@ -0,0 +1,46 @@
+// src/components/TaskForm.tsx
+import React, { useState } from 'react';
+import "../styles/TaskForm.css"
+interface TaskFormProps {
+    onSubmit: (text: string, category: string, priority: 'High' | 'Medium' | 'Low') => void;
+    initialValues?: { text: string; category: string; priority: 'High' | 'Medium' | 'Low' };
```

Comment:
priority is used at different places, Probably add a type/enum and reuse across

Reply:
fixed